src/nxdoc/nuxeo-server/administration/maintenance/nuxeo-admin-console.md

```diff
@@ -43,13 +43,19 @@ Once installed, the Nuxeo admin console can be accessed through a dedicated URL:
 
 ![]({{file name='admin-console-access-refused.png'}} ?border=true)
 
+### Warning on login
+
+A warning popup on login cautions users about tools potentially impacting application performance and can be disabled to prevent future displays.
+
+![]({{file name='admin-console-warning.png'}} ?border=true)
+
 ### Homepage
 
 The homepage provides a quick view into the essential information of your Nuxeo instance:
 * Probes and their status to see if all components are running
 * Version information and cluster enablement information
 
-![]({{file name='admin-console-homepage.png'}} ?border=true)
```

Comment:
Rename admin-console-confirm-reindex.png to admin-console-confirm-elasticsearch-reindex.png